packages/block-directory/src/plugins/get-install-missing/index.js

```diff
@@ -88,7 +88,7 @@ const ModifiedWarning = ( { originalBlock, ...props } ) => {
 			originalBlock.title || originalName
 		);
 		actions.push(
-			<Button key="convert" onClick={ convertToHTML } isLink>
+			<Button key="convert" onClick={ convertToHTML } variant="link">
```

Comment:
Wonder if it's slightly odd to see `link` as a variant. Would it ever make sense to have a "primary link" or a "secondary link"? Should `externalLink` be considered a variant as well (it exists as a separate thin now)?

Reply:
`variant="link"` doesn't determine whether the component renders an `anchor` tag, that's determined by the `href` prop's presence.

Furthermore, the link variant cannot be mixed with the other three variants for a good looking experience. These are the three variants together with `link`:

<img width="113" alt="Captura de Tela 2021-05-21 às 06 48 34" src="https://user-images.githubusercontent.com/24264157/119147830-cbfc2d80-ba00-11eb-8072-c573f5716d25.png">
<img width="114" alt="Captura de Tela 2021-05-21 às 06 48 22" src="https://user-images.githubusercontent.com/24264157/119147832-cc94c400-ba00-11eb-9dd6-81c744846442.png">
<img width="109" alt="Captura de Tela 2021-05-21 às 06 48 04" src="https://user-images.githubusercontent.com/24264157/119147835-cc94c400-ba00-11eb-8d13-babeec4b715a.png">

Reply:
If it's determined by `href` why do we need it as a variant?

Reply:
`link` is a variant of the _style_ of the button, not the functionality. Passing `href` changes the functionality.

Reply:
We should probably rename it to `text` or `flat` though. [`material-ui`](https://material-ui.com/components/buttons/) is a good source of inspiration for this.

Reply:
I'd expect `text` to look most like the variantless button though, i.e., without the underline. 🤷‍♀️ I'm fine renaming it to whatever makes the most sense to people but I don't think we'll find something perfect. Our variants aren't as "varied" as material's. We really just have the five: `no variant`, `primary`, `secondary`, `tertiary`, and `link`. And they all or remove the outline/background arbitrarily (`link`, incidentally, is probably the easiest to guess what it does currently aside from the confusion around functionality).

Reply:
I agree. I think it would be best to merge this change (i.e. introducing variant as a preferred alternative to the overlapping `is` props) and then we could potentially reconsider renaming or changing variants separately. Don't want to keep this PR open any longer because it touches so many usages of Button that it needs constant rebasing, etc.

Reply:
It's weird to use "link" to refer to an appearance characteristic but we can revisit this separately as we should also clarify the design expectations.